e2e_tests/support/commands.ts

```diff
@@ -8,10 +8,21 @@ Cypress.Keyboard.defaults({
 const userName = Cypress.env('USERNAME')
 const password = Cypress.env('PASSWORD')
 
+const userNameSpo = Cypress.env('USERNAME_SPO')
+const passwordSpo = Cypress.env('PASSWORD_SPO')
+
 Cypress.Commands.add('visitPage', url => {
   cy.visit(url)
   cy.pageHeading().should('equal', 'Sign in')
   cy.get('#username').type(userName, { log: false })
   cy.get('#password').type(password, { log: false })
   cy.get('#submit').click()
 })
+
+Cypress.Commands.add('visitPageUser2', url => {
```

Comment:
Can we just reuse `visitPageUser` and pass a second parameter with the login creds?